balancer-js/src/modules/data/pool-joinExit/repository.ts

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any */
+import { PoolJoinExit, PoolJoinExitAttributes } from './types';
+import { BalancerSubgraphRepository } from '@/modules/subgraph/repository';
+import {
+  JoinExit_OrderBy,
+  OrderDirection,
+  SubgraphJoinExitFragment,
+} from '@/modules/subgraph/generated/balancer-subgraph-types';
+
+export class PoolJoinExitRepository extends BalancerSubgraphRepository<
+  PoolJoinExit,
+  PoolJoinExitAttributes
+> {
+  async query(args: any): Promise<PoolJoinExit[]> {
```

Comment:
by default we cache query requests on repositories, how that would be useful for joinExits?

Reply:
it depends on how you want to use the IL. 
I cannot foresee its usage.

Please, could you help in disabling the cache, if needed?

Reply:
in other repository classes caching is actually built-in. reason was to save on needless queries when we use the same data in different services. this caching on data fetching is something we want to refactor anyways and handle it separately. here is all good, because there is no caching. i was just thinking if there would be a need for caching, but true, depends on the use-case, thanks